src/main/java/org/cactoos/iterable/Filtered.java

```diff
@@ -78,7 +78,7 @@ public Filtered(final Func<X, Boolean> fnc, final Iterable<X> src) {
      * @param src Source iterable
      * @param fnc Predicate
      */
-    public Filtered(final Iterable<X> src, final Func<X, Scalar<Boolean>> fnc) {
+    public Filtered(final Iterable<X> src, final Func<? super X, Scalar<Boolean>> fnc) {
```

Comment:
@andreoss we are missing the `? extends X` on the src parameter here

Reply:
@victornoel Done